modules/file-system/parallelstore/variables.tf

```diff
@@ -0,0 +1,81 @@
+/**
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+variable "project_id" {
+  description = "Project in which the HPC deployment will be created."
+  type        = string
+}
+
+variable "deployment_name" {
+  description = "Name of the HPC deployment."
+  type        = string
+}
+
+variable "instance_id" {
+  description = "Name of parallelstore instance."
+  type        = string
+  default     = "parallelstore-instance"
+}
+
+variable "zone" {
+  description = "Location for parallelstore instance."
+  type        = string
+}
+
+variable "region" {
+  description = "The region in which HPC deployment will be created."
+  type        = string
+}
+
+variable "capacity_gib" {
+  description = "Storage size of the parallelstore instance in GB."
+  type        = number
+  default     = 12000
+}
+
+variable "labels" {
+  description = "Labels to add to parallel store instance."
+  type        = map(string)
```

Comment:
Please add `default = {}` to improve usability outside of toolkit )

Reply:
Its not updated https://github.com/GoogleCloudPlatform/hpc-toolkit/blob/main/modules/file-system/filestore/variables.tf#L103 

Will we update this everywhere ?

Reply:
It's low priority, no need to update it everywhere.

Reply:
Done.